src/io/box_filtering.py

```diff
@@ -6,23 +6,31 @@
 1. skip all boxes before 0.5s (before we assume it is unlikely you have sufficient historic)
 2. filter all boxes whose diagonal <= min_box_diag**2 and whose side <= min_box_side
 
+
+
 Copyright: (c) 2019-2020 Prophesee
 """
 from __future__ import print_function
 import numpy as np
 
 
-def filter_boxes(boxes, skip_ts=int(1e5), min_box_diag=30, min_box_side=10):
+def filter_boxes(boxes, skip_ts=int(5e5), min_box_diag=30, min_box_side=10):
     """Filters boxes according to the paper rule. 
 
+    To note: the default represents our threshold when evaluating at VGA resolution (640*360)
```

Comment:
the default value (30) is for VGA or QGVA (aka Gen1)?

Reply:
well, it is a bit unclear, perhaps i should remove this default? i know that for Gen4 we use 60, 20, and imerit i think we use 30,10 (but we also used 30,10 for half resolution of Gen4...)